media-libs/libdiscid/patches/libdiscid-0.6.2.patchset

```diff
@@ -0,0 +1,180 @@
+From b15b936bd13b2d0fc1e280a55c8e8dd3e9839714 Mon Sep 17 00:00:00 2001
+From: sfanxiang <sfanxiang@gmail.com>
+Date: Mon, 15 Jan 2018 02:01:29 +0000
+Subject: port to Haiku
+
+
+diff --git a/Makefile.am b/Makefile.am
+index 31dce4c..dfabcd5 100644
+--- a/Makefile.am
++++ b/Makefile.am
+@@ -77,9 +77,10 @@ libdiscid_la_SOURCES = src/base64.c src/sha1.c src/disc.c
+ libdiscid_la_LDFLAGS = -version-number @libdiscid_VERSION_LT@ -no-undefined
+ libdiscid_la_LIBADD =
+ 
+-#if OS_BEOS
+-#libdiscid_la_LIBADD += -lbe -lroot
+-#endif
++if OS_HAIKU
++libdiscid_la_LIBADD += -lbe -lroot
++libdiscid_la_SOURCES += src/toc.c src/unix.c src/disc_haiku.c
++endif
+ if OS_DARWIN
+ libdiscid_la_LDFLAGS += -framework CoreFoundation -framework IOKit
+ libdiscid_la_SOURCES += src/toc.c src/unix.c src/disc_darwin.c
+diff --git a/configure.ac b/configure.ac
+index 0d21232..accef71 100644
+--- a/configure.ac
++++ b/configure.ac
+@@ -45,7 +45,7 @@ AC_DEFINE(libdiscid_EXPORTS, [], "the header is used to create the library")
+ 
+ 
+ case "${host_cpu}-${host_os}" in
+-  #*-beos*)    os=beos;;
++  *-haiku*)    os=haiku;;
+   *-mingw*)   os=win32;;
+   *-cygwin*)  os=win32;;
+   *-freebsd*) os=freebsd ;;
+@@ -63,7 +63,7 @@ case "${host_cpu}-${host_os}" in
+ esac
+ 
+ AC_MSG_NOTICE([Using disc read implementation for: $os])
+-#AM_CONDITIONAL([OS_BEOS], [test x$os = xbeos])
++AM_CONDITIONAL([OS_HAIKU], [test x$os = xhaiku])
+ AM_CONDITIONAL([OS_DARWIN], [test x$os = xdarwin])
+ AM_CONDITIONAL([OS_FREEBSD], [test x$os = xfreebsd])
+ AM_CONDITIONAL([OS_GENERIC], [test x$os = xgeneric])
+diff --git a/src/disc_haiku.c b/src/disc_haiku.c
+new file mode 100644
+index 0000000..6c07574
+--- /dev/null
++++ b/src/disc_haiku.c
+@@ -0,0 +1,125 @@
```

Comment:
```diff
-index 0000000..6c07574
+index 0000000..58f6ade
 --- /dev/null
 +++ b/src/disc_haiku.c
-@@ -0,0 +1,125 @@
+@@ -0,0 +1,126 @@
```
(Yep, I know. You don't need to make this change because `haikuporter -b` will generate this for you.)